gst/datarepo/gstdatareposrc.c

```diff
@@ -1305,11 +1254,11 @@ gst_data_repo_src_read_json_file (GstDataRepoSrc * src)
   caps_str = json_object_get_string_member (object, "gst_caps");
   GST_INFO_OBJECT (src, "caps_str : %s", caps_str);
 
-  get_caps = gst_caps_from_string (caps_str);
-  GST_INFO_OBJECT (src, "gst_caps : %" GST_PTR_FORMAT, get_caps);
+  src->caps = gst_caps_from_string (caps_str);
```

Comment:
Please handle the case where src->caps is already set. (don't make it leaked)

Reply:
Thank you so much for finding the error.
I will fix this.